tools/offline_log_viewer/controller.py

```diff
@@ -830,6 +873,37 @@ def decode_bootstrap_cluster_cmd_data(rdr: Reader, version):
     if version >= 1:
         decoded |= {'founding_version': rdr.read_int64()}
 
+    # not clear when the following fields where introduced, take a conservative approach and decode them only in version 3
+    if version >= 3:
```

Comment:
I don't understand this comment. We have all the code and the history, don't we know under which version anything was introduced?

Reply:
fixed, found out when initlia_nodes was introduced